airbyte-integrations/connectors/source-punk-api/source_punk_api/punk_api.yaml

```diff
@@ -0,0 +1,46 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.punkapi.com/v2/"
+    http_method: "GET"
+
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  base_stream:
+    schema_loader:
+      type: JsonSchema
+      file_path: "./source_punk_api/schemas/{{ options['name'] }}.json"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
```

Comment:
The API allows using before and after parameters, this allow to implement incremental syncs. Please check the DateStreamSlicer component.